2024-RIKEN-AWS/JupyterNotebook/tutorial/notebook/flux.ipynb

```diff
@@ -1623,7 +1623,7 @@
    "source": [
     "![https://flux-framework.org/flux-operator/_static/images/flux-operator.png](https://flux-framework.org/flux-operator/_static/images/flux-operator.png)\n",
     "\n",
-    "> See you next year! 👋️😎️"
+    "<!-- >> See you next year! 👋️😎️ -->"
```

Comment:
This is hidden now?

Reply:
Yeah, just a minor thing, but since I'm not sure if we'll be giving this tutorial at JLESC again next year, I didn't think it made sense to keep for this version of the tutorial.

Reply:
I also just realized that `dyad.ipynb` is still here. I definitely want to delete that notebook. I'll make a commit for that in a sec

Reply:
> Actually, @vsoch, I just realized that this change was in flux.ipynb. Part of the plan for this tutorial was to break that down into sections (indicated by the notebooks starting with numbers). All the content from flux.ipynb should already be in those notebooks, so do you think it's fine just to delete flux.ipynb?

I haven't tested it out, so I can't say, but I would rely on your judgment - if you can confidently say nothing is lost it's OK to delete. I'll be running it locally (building now) and I can try to check too, but absence of something is often hard to detect!

Reply:
That's a good point.

For me, the biggest thing is to not have it seen by the attendees of the tutorial because I feel like that will get confusing. I could always remove it in `Dockerfile.spawn`. That way we still have the file on GitHub, but it won't be there to confuse the attendees.

Reply:
What do you think, @vsoch?